pyuvsim/simsetup.py

```diff
@@ -39,8 +39,17 @@ def _parse_layout_csv(layout_csv):
         str_format_code = 'a'
     else:
         str_format_code = 'U'
-    dt = np.format_parser([str_format_code + '10', 'i4', 'i4', 'f8', 'f8', 'f8'],
-                          ['name', 'number', 'beamid', 'e', 'n', 'u'], header)
+
+    # get data types for each column
+    dtypes = {"name": str_format_code + '10', 'number': 'i4', 'beamid': 'i4', 'e': 'f8', 'n': 'f8', 'u': 'f8'}
```

Comment:
We should probably mention somewhere that the antenna names in layout files are limited to 10 characters.